src/QsCompiler/Compiler/RewriteSteps/MonomorphizationRewriteStep.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Quantum.QsCompiler;
+using Microsoft.Quantum.QsCompiler.SyntaxTree;
+using Microsoft.Quantum.QsCompiler.Transformations.Monomorphization;
+using Microsoft.Quantum.QsCompiler.Transformations.MonomorphizationValidation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Compiler.RewriteSteps
+{
+    internal class MonomorphizationRewriteStep : IRewriteStep
+    {
+        public string Name { get; }
+
+        public int Priority { get; }
+
+        public string OutputFolder { get; set; }
+
+        public bool ImplementsTransformation { get; }
+
+        public bool ImplementsPreconditionVerification { get; }
+
+        public bool ImplementsPostconditionVerification { get; }
+
+        public MonomorphizationRewriteStep(bool doPostConditionValidation = false)
```

Comment:
It seems like you added basically an override to disable the post condition verification in all cases here? What is the reason for that?